bot/exts/moderation/infraction/infractions.py

```diff
@@ -280,8 +280,19 @@ async def unvoiceban(self, ctx: Context, user: FetchedMember) -> None:
 
     async def apply_mute(self, ctx: Context, user: Member, reason: t.Optional[str], **kwargs) -> None:
         """Apply a mute infraction with kwargs passed to `post_infraction`."""
-        if await _utils.get_active_infraction(ctx, user, "mute"):
-            return
+        if active := await _utils.get_active_infraction(ctx, user, "mute", send_msg=False):
+            if active["actor"] != self.bot.user.id:
+                await _utils.send_active_infraction_message(ctx, active)
+                return
+
+            # Let the current mute attempt override an automatically triggered mute.
```

Comment:
This wording is a bit ambiguous, is "attempt" in this context a noun or verb?

As in, "Let the current attempt to mute the user override an automatically triggered mute." or is it "Let the current mute attempt to override an automatically triggered mute."?

I prefer some variation of the former, I think it could be improved.

Reply:
I just forgot to write "to" after "attempt".

Reply:
Well, I didn't forget. I was using it as a noun, but I agree it's ambiguous.